assets/xml/objects/object_famos.xml

```diff
@@ -1,28 +1,30 @@
 ﻿<Root>
     <File Name="object_famos" Segment="6">
-        <Animation Name="object_famos_Anim_0000F8" Offset="0xF8" />
-        <DList Name="object_famos_DL_0012B0" Offset="0x12B0" />
-        <DList Name="object_famos_DL_001528" Offset="0x1528" />
-        <DList Name="object_famos_DL_001C88" Offset="0x1C88" />
-        <DList Name="object_famos_DL_001DE8" Offset="0x1DE8" />
-        <DList Name="object_famos_DL_001FA8" Offset="0x1FA8" />
+        <Animation Name="gFamosShakeAnim" Offset="0xF8" />
+        <DList Name="gFamosHelmetDL" Offset="0x12B0" />
+        <DList Name="gFamosMainBodyDL" Offset="0x1528" />
+        <DList Name="gFamosShieldDL" Offset="0x1C88" />
+        <DList Name="gFamosSwordDL" Offset="0x1DE8" />
+        <DList Name="gFamosLightSwitchDL" Offset="0x1FA8" />
         <Texture Name="object_famos_TLUT_002068" OutName="tlut_002068" Format="rgba16" Width="4" Height="4" Offset="0x2068" />
         <Texture Name="object_famos_TLUT_002088" OutName="tlut_002088" Format="rgba16" Width="4" Height="4" Offset="0x2088" />
         <Texture Name="object_famos_TLUT_0020A8" OutName="tlut_0020A8" Format="rgba16" Width="4" Height="4" Offset="0x20A8" />
         <Texture Name="object_famos_TLUT_0020C8" OutName="tlut_0020C8" Format="rgba16" Width="4" Height="4" Offset="0x20C8" />
```

Comment:
Is there any pattern to how these are used? If not can just name them `gFamos0TLUT` etc.